spec/services/placement_export_csv_service_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+require "factory_bot_rails"
+
+RSpec.describe PlacementExportCsvService do
+  let(:casa_org) { create(:casa_org, name: "Fake Name", display_name: "Fake Display Name") }
+  let(:placement_type) { build(:placement_type, casa_org: casa_org) }
+  let(:creator) { build(:user) }
+  let(:placement) { build(:placement, creator: creator, placement_type: placement_type) }
```

Comment:
@rae-stanton would you mind just defining all of these within the `it` block instead of using `let`? It makes the specs easier to follow.